features/hooks/before_and_after_hooks.feature

```diff
@@ -300,6 +300,31 @@ Feature: `before` and `after` hooks
       after suite
       """
 
+  Scenario: `after` blocks are run also for examples which have failed or encountered other errors
+    Given a file named "after_example_spec.rb" with:
+      """ruby
+      RSpec.describe "after callbacks" do
+        after(:example) do
+          puts "this is executed"
+          raise "error in after example block"
+        end
+
+        after(:example) do
+          puts "this is also executed"
+          raise "another error"
+        end
+
+        example "deliberate failure" do
+          expect(true).to be(false)
```

Comment:
It's pretty obvious that after hooks are being run for failed examples.

I believe the original issue had in mind an example that raises an exception:
```ruby
  it do
    raise "in example"
  end
```

Do you mind to change this feature to reflect that scenario?

For completeness, it would be nice to have scenarios when the example is pending or skipped.